modules/calendar/spec/features/calendar_widget_spec.rb

```diff
@@ -50,15 +50,19 @@
   let(:wp_full_view) { Pages::FullWorkPackage.new(work_package, project) }
   let(:calendar) { Pages::Calendar.new project }
 
-  current_user do
+  shared_let(:current_user) do
```

Comment:
There is now a `shared_current_user` which does the same as `current_user`, but with `shared_let`.

Reply:
The problem with these helpers is that they create a new `before` block, which changes the execution. I found this is weird if you multiple of these

Reply:
good point.